tests/common/models.py

```diff
@@ -396,3 +396,15 @@ def configure_tiny_bert_config():
 
 def configure_tiny_bert_hf_model(use_logits=True):
```

Comment:
Maybe add a comment here similar to the one where pytest.tiny_gpt_model is defined, explaining why we use this, so it's clear in both places?

Reply:
yup, makes sense. will do